Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -268,7 +268,8 @@ - (void)updateNearby:(nullable dispatch_block_t)completion {
 
     [group waitInBackgroundWithTimeout:WMFFeedRefreshTimeoutInterval
                             completion:^{
-                                WMFAssertMainThread(@"completion must be called on the main thread");                                self.nearbyUpdateTaskGroup = nil;
+                                WMFAssertMainThread(@"completion must be called on the main thread");
+                                self.nearbyUpdateTaskGroup = nil;
```

Comment:
This is not an addition, a just reformat. The `self.nearbyUpdateTaskGroup = nil;` was previously on the same line as the assert.